AllASPFuncs.py

```diff
@@ -598,34 +598,20 @@ def constructImages(exposures, ra, dec, size=7, background=False,
         '''
 
         # If we are not fitting the background we manually subtract it here.
-        if background:
-            # When background is true, we are including the background level as
+        if fit_background:
+            # When fit_background is true, we are including the background level as
             # a free parameter in our fit, so it should not be subtracted here.
             bg = 0
-        elif not background and not truth == 'truth':
+        elif not fit_background and not truth == 'truth':
```

Comment:
1. I would nest this if, so that the `if truth` cases are just within the `elif not fit_background` block.
2. I would also just skip the `if fit_background:` part and replace that with set `bg = 0`, and then `if not fit_background:`.